src/aspects/ec2-automated-shutdown/handler/index.ts

```diff
@@ -0,0 +1,131 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Context } from 'aws-lambda';
+import { EC2Client, StopInstancesCommand } from '@aws-sdk/client-ec2';
+
+interface SNSEvent {
```

Comment:
This should already exist from the `aws-lambda` library. Should be a drop in replacement:

```
import { SNSEvent } from 'aws-lambda'
```